packages/headless/src/features/advanced-search-queries/advanced-search-queries-slice.test.ts

```diff
@@ -56,7 +56,7 @@ describe('advanced search parameters', () => {
       aq: 'hola',
     };
     const historyChange = {
-      ...getHistoryEmptyState(),
+      ...getHistoryInitialState(),
```

Comment:
This isn't just a renaming, the function was moved. I figured I would also rename it to match other `*InitialState()` methods.